src/test/resources/hudson/plugins/parameterizedtrigger/test/BuildInfoExporterTest/testMigrateFrom221/jobs/upstream/config.xml

```diff
@@ -104,11 +102,10 @@
               <color>RED</color>
             </failureThreshold>
           </block>
-          <buildAllNodesWithLabel>false</buildAllNodesWithLabel>
```

Comment:
In this test change you hide the problem you introduce by breaking the compatibility